### PR TITLE
Feature/express

### PR DESCRIPTION
**Pull request by JainishaParmar**
changes in files

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
done

**Review by jaimin45 — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Review by jaimin45 — APPROVED** [review]

File: .eslintrc.json [hunk 1]
```diff
@@ -8,10 +8,25 @@
       "ecmaVersion": 2021,
       "sourceType": "module"
   },
-  "plugins": ["prettier", "spellcheck"],
+  "plugins": ["prettier", "spellcheck","import"],
+  // "settings": {
```
Comment (kapilbarad) [inline .eslintrc.json]: Please remove this commented code.

File: app.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+mangoose
+  .connect("mongodb://localhost:27017/tutorials", {
+    useNewUrlParser: true,
+  })
+  .then(() => console.log("Connection successfull!"))
+  .catch((err) => console.log);
+
+// Schema
+const tutorialSchema = new mangoose.Schema({
```
Comment (kapilbarad) [inline app.js]: Define this model ina seperate file.
Example:
```
app\
  routes\
  models\
     Tutorial.js # Define schema here
     index.js # export all the models from here using module.exports = { Tutorial }
```

CC: @JainishaParmar @jaimin45 @ashishkava

**Review by kapilbarad — CHANGES_REQUESTED** [review]

File: app.js [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+const swaggerUI = require('swagger-ui-express');
+const swaggerYaml = require('yamljs');
+
+const indexRouter = require('./routes/index');
+const usersRouter = require('./routes/users');
+
+const dirname = path.resolve();
+
+mangoose
```
Comment (kapilbarad) [inline app.js]: This should be in a seperate folder
```
app\ 
   init\
     db.js # export default the connection
```

File: app.js [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+    });
+
+    const result = await demoTutorial.save();
+    console.log(result);
+  } catch (err) {
+    console.log(err);
+  }
+};
+
+createDocument();
```
Comment (kapilbarad) [inline app.js]: Please remove this createDocument and createDocument(); That will be next task when we will implement controllers.

**Review by kapilbarad — CHANGES_REQUESTED** [review]

**Review by jaimin45 — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Comment by jaimin45** [comment]
feat: Features/Express-Generator

**Pull request merged**